app/src/pages/inside/logsPage/logItemInfo/logItemDetails/logItemDetails.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { DurationBlock } from 'pages/inside/common/itemInfo/durationBlock/';
```

Comment:
Please remove the trailing slash from the path

Reply:
I guess my IDE put this slash by some reason.